src/coreclr/jit/lsra.cpp

```diff
@@ -9613,13 +9634,17 @@ void LinearScan::TupleStyleDump(LsraTupleDumpMode mode)
                 // and combining the fixed regs with their associated def or use
                 bool         killPrinted        = false;
                 RefPosition* lastFixedRegRefPos = nullptr;
-                for (; refPosIterator != refPositions.end() &&
-                       (currentRefPosition->refType == RefTypeUse || currentRefPosition->refType == RefTypeFixedReg ||
-                        currentRefPosition->refType == RefTypeKill || currentRefPosition->refType == RefTypeDef) &&
-                       (currentRefPosition->nodeLocation == tree->gtSeqNum ||
-                        currentRefPosition->nodeLocation == tree->gtSeqNum + 1);
-                     ++refPosIterator, currentRefPosition = &refPosIterator)
+                for (; refPosIterator != refPositions.end(); ++refPosIterator)
                 {
+                    currentRefPosition = &refPosIterator;
+                    if (!(currentRefPosition->refType == RefTypeUse || currentRefPosition->refType == RefTypeFixedReg ||
+                          currentRefPosition->refType == RefTypeKill || currentRefPosition->refType == RefTypeDef) ||
+                        !(currentRefPosition->nodeLocation == tree->gtSeqNum ||
+                          currentRefPosition->nodeLocation == tree->gtSeqNum + 1))
```

Comment:
I think this one would read better as
```
                    if ((currentRefPosition->refType != RefTypeUse && currentRefPosition->refType != RefTypeFixedReg &&
                         currentRefPosition->refType != RefTypeKill && currentRefPosition->refType != RefTypeDef) ||
                        (currentRefPosition->nodeLocation != tree->gtSeqNum &&
                         currentRefPosition->nodeLocation != tree->gtSeqNum + 1))
```

Reply:
Can you address this in a follow-up? I'd prefer to avoid retriggering the CI especially if you simply going to go back through this.